.github/workflows/sync-labels.yaml

```diff
@@ -9,6 +9,7 @@ on:
 
 jobs:
   label-sync:
+    if: github.repository == 'scylladb/scylladb'
```

Comment:
just to be on the safe side, might want to use
```yaml
if: ${{ github.repository == 'scylladb/scylladb' }}
```

Reply:
thanks. i understand this is exactly the example used in https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution#overview . but we had an interesting case where a conditional step was not triggered. hence this suggestion.